doc/schema/attribute-types/text.md

```diff
@@ -13,43 +13,69 @@ export const Todo: PartialSchema = {
 
 ## Parameters
 
-- `default` [{String}] - The default value of the attribute.  Example: `text({default: "hello"})`
-- `required` [{Boolean=false}] - If the attribute must be provided.
+|      key      | description                                                                 | type            | optional  | default |
+| ------------- | --------------------------------------------------------------------------- | :-------------: | :-------: | :-----: |
+| `default`     | The default value of the attribute. <br/> Example: `text({default: true})`  | `Boolean`       | Yes       |         |
+| `required`    | If the attribute must be provided.  <br/> Example: `text({required: true})` | `Boolean`       | Yes       | `false` |
 
-## Form Controls
+## Database and Sequelize Behavior
 
-`text()` will produce a [`<textarea>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) control. 
+The `text` type will create a sequelize [DataTypes.TEXT](https://sequelize.org/docs/v6/core-concepts/model-basics/#strings) column.
 
+## Middleware Behavior
 
-<details>
-<summary>
+### Querying Data
 
-## Advanced Details
+For text, use any string value in your queries as follows:
 
-</summary>
+```js
+GET /todos?filter[notes][$eq]=this%20is%20super%20important //all todos with notes that equal "this is super important"
+GET /todos?filter[notes][$eq]=null //all todos with notes that are null
```

Comment:
not sure if we should document this, or the future behaviour and put a 🛑 emoji. this iteration will return `notes` that are `null` or `"null"`?